lib/ometajs/core.js

```diff
@@ -443,7 +449,10 @@
 		_disablePrependingInput: function() {
 			this._applyWithArgs = function(rule) {
 				var origInput = this.input,
-					ruleArgs = arguments.length > 1 ? __slice.call(arguments, 1) : [];
+					ruleArgs = new Array(arguments.length - 1);
```

Comment:
what happens if arguments.length == 0?
previously it would result in an empty ruleArgs, now it will throw RangeError